js/main.js

```diff
@@ -37,13 +38,13 @@ Mousetrap.bind(['command+c', 'ctrl+c'], function() {
 			selectedCubelet = activeNode.cubelet;
 		}
 	}
-	reGenerate();
+	regenerate();
 	return false;
 });
 
 //shortcut to add a node
-Mousetrap.bind(['command+a', 'ctrl+a', 'plus'], function() {
```

Comment:
Why have you removed 'ctrl+a'? I use this when I'm navigating the tree. Keeping it isn't a bad thing.